src/routes/Header.svelte

```diff
@@ -2,16 +2,54 @@
   import { page } from "$app/stores";
   import logo from "$lib/images/svelte-logo.svg";
   import github from "$lib/images/github.svg";
+  import "iconify-icon";
+
+
+  $: current = $page.url.pathname
+
 </script>
 
-<header>
-  <div class="corner">
-    <a href="https://kit.svelte.dev">
-      <img src={logo} alt="SvelteKit" />
-    </a>
-  </div>
+<header class="navbar bg-base-100">
+    <div class="navbar-start">
+      <a href='/' class="btn btn-ghost normal-case text-xl">Planet Of The Bugs</a>
+    </div>
+
+    <ul class="navbar-center hidden lg:flex justify-between items-center w-1/3 text-lg font-bold">
+      <li 
+      class={current === '/' ?
+       "hidden lg:flex transition-all underline-offset-8 decoration-primary hover:underline underline"
+        : "hidden lg:flex transition-all underline-offset-8 decoration-primary hover:underline"}
+        >
+        <a href="/">Home</a>
+      </li>
+      <li 
+      class={current === '/courses' ?
+       "hidden lg:flex transition-all underline-offset-8 decoration-primary hover:underline underline"
+        : "hidden lg:flex transition-all underline-offset-8 decoration-primary hover:underline"}
+        >
+        <a href="/courses">Courses</a>
+      </li>
+      <li 
+      class={current === '/about' ?
+       "hidden lg:flex transition-all underline-offset-8 decoration-primary hover:underline underline"
+        : "hidden lg:flex transition-all underline-offset-8 decoration-primary hover:underline"}
+        >
+        <a href="/about">About</a>
+      </li>
+    </ul>
+
+    <div class="navbar-end">
+       <button class="lg:hidden block text-3xl btn-ghost">
+        <iconify-icon icon="heroicons:bars-3-bottom-right" />
+       </button>
+
+       <button class="lg:block hidden text-3xl btn-ghost mr-4">
+        <iconify-icon icon="heroicons:user-solid" />
```

Comment:
Looks great so far! Regarding the user icon, how about we use an avatar? 
Something a bit similar to this
https://daisyui.com/components/navbar/#navbar-with-icon-indicator-and-dropdown
<img width="883" alt="Screenshot 2023-04-01 at 20 18 16" src="https://user-images.githubusercontent.com/21015204/229305275-2a65b62b-00aa-4fb6-91d2-a2d362c0b383.png">

Reply:
yeah i planned to do so after this structure commit